installer/templates/phx_web/router.ex

```diff
@@ -28,21 +37,15 @@ defmodule <%= web_namespace %>.Router do
   # If your application does not have an admins-only section yet,
   # you can use Plug.BasicAuth to set up some basic authentication
   # as long as you are also using SSL (which you should anyway).
-  if Mix.env() == :dev do
-    import Phoenix.LiveDashboard.Router
+  if Mix.env() in [:dev, :test] do
```

Comment:
Unfortunately, this means if someone deploys a new app with the layout as is, the production version won't work. My suggestion is to keep this dev only and do `<%= if function_exported?(Routes, :live_dashboard_path, 2) do %>` in the layout.

Reply:
Updated.  This leads to the following warning in non-dev environments:

```
warning: DeadAppWeb.Router.Helpers.live_dashboard_path/2 is undefined or private
```

Is this acceptable, or can/should I do more to silence the warning?

Reply:
Let's silence the warning in test and we are good. I dropped a suggestion.

Reply:
Done, thanks!